qa-core/src/internal-api/tests/dataSources/postgresSQL.spec.ts

```diff
@@ -0,0 +1,71 @@
+import TestConfiguration from "../../config/TestConfiguration"
+import * as fixtures from "../../fixtures"
+
+describe("Internal API - Data Sources: PostgresSQL", () => {
+  const config = new TestConfiguration()
+
+  beforeAll(async () => {
+    await config.beforeAll()
+  })
+
+  afterAll(async () => {
+    await config.afterAll()
+  })
+
+  it("Create an app with a data source - PostgresSQL", async () => {
+    // Create app
+    await config.createApp()
+
+    // Add data source
+    const [dataSourceResponse, dataSourceJson] =
+      await config.api.datasources.add(fixtures.datasources.postgresSQL())
+
+    // Update data source
+    const newDataSourceInfo = {
+      ...dataSourceJson.datasource,
+      name: "PostgresSQL2",
+    }
+    const [updatedDataSourceResponse, updatedDataSourceJson] =
+      await config.api.datasources.update(newDataSourceInfo)
+
+    const dataSourceQuery = {
```

Comment:
For this and other `dataSourceQuery` vars we could probably pull out to a `fixtures.queries` helper as there looks to be some overlap. We could then use like

```ts
fixures.queries.query(datasourceId, fields)
```

could even add some nice helpers like
```ts
fixures.queries.sqlQuery(datasourceId, "SELECT * FROM categories;")
```